mkdocs.yml

```diff
@@ -96,3 +95,6 @@ plugins:
       on_pre_build: 'docs.plugins.main:on_pre_build'
       on_files: 'docs.plugins.main:on_files'
       on_page_markdown: 'docs.plugins.main:on_page_markdown'
+- redirects:
+    redirect_maps:
+      'usage/mypy.md': 'mypy_plugin.md'
```

Comment:
the lack of newline here is causing linting to fail.

Good catch on redirects :+1: